fineract-provider/src/main/java/org/apache/fineract/commands/handler/NewCommandSourceHandler.java

```diff
@@ -20,8 +20,12 @@
 
 import org.apache.fineract.infrastructure.core.api.JsonCommand;
 import org.apache.fineract.infrastructure.core.data.CommandProcessingResult;
+import org.springframework.transaction.annotation.Isolation;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
 
 public interface NewCommandSourceHandler {
 
+    @Transactional(propagation = Propagation.REQUIRES_NEW, isolation = Isolation.REPEATABLE_READ)
```

Comment:
I don't remember. What if the handler implementor also uses @Transactional? Does it override this behavior or not?